app/views/oauth2_applications/index.html.erb

```diff
@@ -2,12 +2,22 @@
   <h1><%= t ".title" %></h1>
 <% end %>
 
+<% if @applications.length > 0 %>
+  <table class="table table-borderless table-striped">
+    <thead>
+      <th><%= t ".name" %></th>
+      <th><%= t ".permissions" %></th>
+      <th></th>
+      <th></th>
+    </thead>
+    <tbody>
+      <%= render :partial => "application", :collection => @applications %>
+    </tbody>
+  </table>
+<% else %>
+  <p><%= t ".no_applications_html", :oauth2 => link_to(t(".oauth_2"), "https://oauth.net/2/") %></p>
```

Comment:
What's the reason for pulling out the link like that? It creates a potential lego issue in the translations and the main translation is already an HTML string anyway - is it just to stop translators changing the link target?

Reply:
It was done this way just to match the corresponding `no_apps_html` from the OAuth (1) client applications page.

Reply:
Ah ok. I just realised there are two versions of `no_applications_html` but only one of `oauth_2` so I thought you might be sharing that but it seems not so I'm wondering if the other one works properly?